draft-ietf-jsonpath-base.md

```diff
@@ -1367,8 +1367,12 @@ Sections {{<match}} and {{<search}} below for details about these.)
 JSON:
 
     {
-      "a": [3, 5, 1, 2, 4, 6, {"b": "j"}, {"b": "k"},
-            {"b": {}}, {"b": "kilo"}],
+      "a": [3, 5, 1, 2, 4, 6,
+            {"b": "j"},
+            {"b": "k"},
+            {"b": {}},
+            {"b": "kilo"}
+           ],
```

Comment:
I cherry-picked the above improvement into https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/455.